lib/analysis_options.yaml

```diff
@@ -194,12 +200,20 @@ linter:
     # - prefer_double_quotes
     - prefer_equal_for_default_values
     # We turned this off due to the fact that we don't want to force the use of this arrow-syntax return when we have a
-    # multiline return. Created an issue https://github.com/dart-lang/linter/issues/2469 in the linter where it might be
-    # actionable after the discussion.
+    # multiline return.
+    #
+    # Created an issue https://github.com/dart-lang/linter/issues/2469 where we briefly discussed the possibility of
+    # requiring this only if not multi-line, but it appears that it wouldn't make much sense given other lints.
+    #
+    # And just like https://dart.dev/guides/language/effective-dart/usage#consider-using--for-simple-members says, it's
+    # a CONSIDERATION, so it must be evaluate per case, and not simply whether it's single or multi line.
     # - prefer_expression_function_bodies
     - prefer_final_fields
     - prefer_final_in_for_each
     - prefer_final_locals
+    # TODO: Unreleased, and not quite sure if we should enforce this, as it gives a TON of verbosity to the overall
```

Comment:
Agreed with your observation. I'm not sure if the benefits justify the verbosity (also, I not even remember if we had this problem someday)